src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -190,24 +196,32 @@ public boolean link() throws IOException, InterruptedException {
 
         ProcessRunner zipAlign = new ProcessRunner(buildToolsPath.resolve("zipalign").toString(), "-f", "4", unalignedApk, alignedApk);
         zipAlign.runProcess("zipalign");
+
         createDevelopKeystore();
+
+        ProcessRunner sign =  new ProcessRunner(buildToolsPath.resolve("apksigner").toString(), "sign", "--ks", 
+        paths.getGvmPath().resolve("debugkeystore.jks").toString(), "--ks-key-alias", "androiddebugkey", "--ks-pass", "pass:android", "--key-pass", "pass:android",  alignedApk);
+        sign.runProcess("sign");
```

Comment:
the `runProcess` call returns 0 if the process ended successfully, so we can use that instead of `return true`?